netbox/netbox/views/generic/bulk_views.py

```diff
@@ -776,6 +776,14 @@ def post(self, request):
                             )
                             return redirect(self.get_return_url(request))
 
+                except IntegrityError as e:
+                    messages.error(self.request, ", ".join(e.args))
+                    clear_events.send(sender=self)
+
+                except ValidationError as e:
+                    messages.error(self.request, ", ".join(e.messages))
+                    clear_events.send(sender=self)
+
```

Comment:
Are these changes strictly relevant to #14044? If not, we should break them out into a separate bug report.

Reply:
They are.  This will throw a validation error if we don't use regex (We can easily dump that though if we want to go to `None == ''`)

The Integrity error will catch any associated integrity errors.  I tried to fine a way to separate the integrity error out and only capture an integrity error that relates to the name field but it wasn't easily do-able.